server/database/models/Imagine25.js

```diff
@@ -0,0 +1,40 @@
+module.exports = (sequelize, DataTypes) => {
+  const Imagine25 = sequelize.define(
+      'Imagine25', {
+        id: {
+          type: DataTypes.INTEGER,
+          unique: true,
+          primaryKey: true,
+          autoIncrement: true,
+        },
+        userid: {
+          type: DataTypes.TEXT,
+        },
+        section: {
+          type: DataTypes.ENUM,
+          values: ['experiential',
+            'discomfortCountPOC',
+            'discomfortCountNonPOC',
+            'control'],
```

Comment:
I think these values should be 'experiential','expression','control' based on the schema.sql file Carla updated